requirements.txt

```diff
@@ -1,6 +1,6 @@
 cement>=3.0.4
 jinja2
-pyyaml==3.13
+pyyaml
```

Comment:
We had pyyaml pinned to 3.13 to force upgrades to old versions of pyyaml that were installed by the system. I think it might be more appropriate to pin to >=3.13.

Reply:
No problem, just updated to >=3.13